.doc.holiday/styleguide-docs.md

```
@@ -0,0 +1,307 @@
+  - [Scheduling](/workflow-management/scheduling/)
+  - [Tables](/reporting-analytics/tables/)
+  - [Actions](/workflow-management/actions/) and [triggers](/workflow-management/triggers/) are two kinds of steps that you use when [building your workflows](/workflow-management/building-workflows/) in OpenOps.
+- Do not use relative links.
+
+**External links**
+- Use full URLs and add a short note when external destinations require credentials or billing.
+- Example: [AWS documentation](https://docs.aws.amazon.com/)
+- Do not use HTML `<a>` tags; use Markdown links only.
+- Do not try to control link targets; Mintlify automatically opens external links in a new tab or window.
+
+**Cross-references and anchors**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)